### ISSUE TITLE
CovPassCheck shows invalid offline status

### ISSUE THREAD

**Issue by timokoenig** [Digitaler-Impfnachweis/covpass-ios#119]
## Avoid duplicates
* [x] Bug is not mentioned in the [FAQ](https://digitaler-impfnachweis-app.de/faq/)
* [x] Bug is not already reported in another issue

## Describe the bug

CovPassCheck shows "Offline mode available" even though the rule set is outdated. This could potentially result in invalid rule checks.

## Steps to reproduce the issue

<img src="https://user-images.githubusercontent.com/4365095/151670434-7a80b2f1-8193-4430-89a8-a155bdad4af4.PNG" width="300" />

1. Wait min. 24 hours before opening the app
2. Open the app, wait until list of certificates has been updated
3. !! After step 2 the bug appears
4. Switch on flight mode
5. The rules are unable to update but the status is green

## Expected behaviour

The offline mode should only be available after the update of the local rule set has been completed.

## Possible Fix

Do not update the status of the offline mode until everything has been downloaded.

**Possible enhancement**: Show the user that something is downloading. If an error occurs the user will not notice and thinks that the download is done or still in progress.

**Comment by alexcimander** [comment]
hey @timokoenig 🏅 ,

thank you for reporting this bug. I created a ticket for it.

**Comment by Kurt57** [comment]
this should be fixed @timokoenig. can you confirm this ?

**Comment by timokoenig** [comment]
I can't confirm it anymore but I will close the issue
